util/dvsim/Deploy.py

```diff
@@ -311,6 +311,38 @@ def get_status(self):
             self.link_odir()
             del self.process
 
+    def kill(self):
+        '''kill running processes
+        '''
+        if self.status == "D" and self.process.poll() is None:
```

Comment:
also include `not self.status` clause (for items not yet dispatched).

Reply:
do you mean to add below? if not yet dispatch, don't need to do anything, right?
if `self.status and` self.status == "D"

Reply:
actually yes, no need to do anything. existing code is fine.

Reply:
OK. Please let me know if you have any other comment